### DISCUSSION TITLE
Edit Titlebar theming

### DISCUSSION THREAD

**Discussion by jelabarre59** [themix-project/oomox-gtk-theme#87] [General]
I was looking at this application to see if I could edit/incorporate an older Metacity theme into the GTK3/4 theming.  I'm concerned that at some poing the GTK folks are going to break the theming for titlebars, and I'll be stuck with the ugly "headers" that mashes the title into a featureless header that can't be differentiated from a window background (and where an active window can't be differentiated from an inactive one).  I'm currently running Cinnamon Desktop (Muffin window manager), but the Cinnamon folks have im0plied that the ability to use Metacity themes may go away, and we'd have to convert those to GTK's CSS-based format.

**Comment by actionless** [comment]
1) you already can edit the background/foreground of the headerbar
2) ... and the border color of the window
3) so basically the only remaining part is this: https://github.com/themix-project/oomox-gtk-theme/issues/58

**Comment by jelabarre59** [comment]
I took another look at this application, and I still don't see a way to edit/configure the TITLEBAR.   The 'headerbar' looks to be the current replacement for the toolbar with all sorts of other junk crammed into it.

I'd like to find a GTK3-native configuration so I don't have to keep using GTK-nocsd, (and it's apparent GTK3/4 will eventually *break* the ability to load old Metacity XML files).  Looking to port https://github.com/charlesbos/metacity-themes/tree/master/Esco/metacity-1 as an example.

**Reply by actionless** [reply]
metacity is there, just check which values you need there: https://github.com/themix-project/oomox-gtk-theme/tree/master/src/metacity-1

**Comment by jelabarre59** [comment]
I went back to this again, and I _still_ don't see a way to re-create the _**TITLEBAR**_ for GTK3.  Note I say TITLEBAR, not those abominable _Headerbars_ the GNOME folks have inflicted upon us.  I can use _gtk3-classic_ along with _gtk3-nocsd_ to get rid of the headerbar, but still can't edit/change the TITLEBAR settings.  

GNOME has broken yet something else for us.

**Reply by actionless** [reply]
there is no such thing as "just titlebar", it's either gtk titlebar, or metacity titlebar or openbox titlebar etc

and yelling that "IT'S TITLEBAR, not titlebar" won't make it more clear for anyone

i know there are such services as "Computer Doctor" where you call and a guy comes to fix your computer

**Comment by jelabarre59** [comment]
_I'm_ the guy people call to fix their computers after the fumbling klutzes at those "Computer Doctor" places completely foul up their computer.
